packages/client/src/components/screens/operations/year-end-operations/index.tsx

```
@@ -0,0 +1,68 @@
+      vacationReserveCharge {
+        id
+      }
+      bankDepositsRevaluationCharge {
+        id
+      }
+    }
+  }
+`;
+
+export const YearEndOperations = (): ReactElement => {
+  const [generateRevaluationCharge] = useMutation(GenerateFinancialChargesDocument);
```

ESLint

[error] 'generateRevaluationCharge' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)